.bazelversion

```diff
@@ -1 +1 @@
-7.4.1
+7.6.0
```

Comment:
```suggestion
7.6.1
```
New version is out and already suggested by renovate https://github.com/edgelesssys/constellation/pull/3733

Reply:
Unfortunately, it's not yet in nixpkgs. We should ensure to stay compatible with what's available in the Nix shell.